boards/arc/hsdk/hsdk.yaml

```diff
@@ -4,7 +4,7 @@ type: mcu
 arch: arc
 toolchain:
   - zephyr
-  - xtools
+  - arcmwdt
```

Comment:
Please do not remove xtools. It allows using custom built GCC toolchain, and is useful when testing builds from Zephyr's sdk-ng repo.

Reply:
OK, added.